internal/handler/msg_find.go

```diff
@@ -110,6 +110,24 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 		qp.Filter = params.Filter
 	}
 
+	if err = common.ValidateSortDocument(params.Sort); err != nil {
```

Comment:
Let's make a function that would validate the give `sort` document and return a canonical form of it: `{"a": int32(1), "b": int32(-1)}`. Then, backends will not have a need to validate it.